### DISCUSSION TITLE
Concept of platforms

### DISCUSSION THREAD

**Discussion by dtslvr** [ghostfolio/ghostfolio#2736] [Q&A]
I just got Ghostfolio running locally and I'm struggling to understand what "Platforms" are and how they are supposed to work. Can you point me to some documentation?
Thanks for your work

_Originally posted by @geeanlooca in https://github.com/ghostfolio/ghostfolio/issues/1957#issuecomment-1847333435_

**Comment by dtslvr** [comment]
Hi @geeanlooca

_Platforms_ is a concept to group multiple accounts. Maybe you have a saving and trading account at the same bank. By assigning the same platform, they are displayed with an icon and you will be informed about platform risks.

**Reply by geeanlooca** [reply]
Thank you @dtslvr!
Understood.
